LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -872,15 +873,17 @@ - (void)_setupOptionsButton {
 }
 
 - (void)_setupPasscodeOverlayBorder {
-    CALayer *topBorder = [CALayer layer];
+    UIView *topBorder = [UIView new];
+    [topBorder setAutoresizingMask:UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleBottomMargin];
     topBorder.frame = CGRectMake(0, 0, self.view.frame.size.width, 1);
-    topBorder.backgroundColor = _textFieldBorderColor.CGColor;
-    [_complexPasscodeOverlayView.layer addSublayer:topBorder];
+    topBorder.backgroundColor = _textFieldBorderColor;
+    [_complexPasscodeOverlayView addSubview:topBorder];
 
-    CALayer *bottomBorder = [CALayer layer];
+    UIView *bottomBorder = [UIView new];
+    [bottomBorder setAutoresizingMask:UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleBottomMargin];
```

Comment:
why don't we use autolayout?

Reply:
autoresizing mask is a very old technology which is replaced by autolayout

Reply:
I tried to add top and bottom border to the UIView, but the method before does not work with autolayout. 
So I found this solution https://stackoverflow.com/a/29033559

Reply:
You need to set `translatesAutoresizingMaskIntoConstraints = false` for all the views to be added by autolayout.

Reply:
updated